app/src/main/java/com/example/benchproject/app/popular/movies/view/PopularMoviesRecyclerAdapter.kt

```diff
@@ -1,42 +1,23 @@
 package com.example.benchproject.app.popular.movies.view
 
-import android.annotation.SuppressLint
 import android.view.LayoutInflater
 import android.view.ViewGroup
-import androidx.recyclerview.widget.RecyclerView
-import com.example.benchproject.app.common.binding.imageView.loadMovieImage
+import com.example.benchproject.app.common.recyclerView.BaseAdapter
+import com.example.benchproject.app.common.recyclerView.BaseViewHolder
 import com.example.benchproject.app.popular.movies.entity.Movie
 import com.example.benchproject.app.popular.movies.viewModel.PopularMoviesViewModel
 import com.example.benchproject.databinding.PopularMovieRecyclerItemLayoutBinding
 
-class PopularMoviesRecyclerAdapter(private val moviesViewModel: PopularMoviesViewModel) : RecyclerView.Adapter<PopularMoviesRecyclerAdapter.PopularMoviesHolder>() {
+class PopularMoviesRecyclerAdapter(private val moviesViewModel: PopularMoviesViewModel) : BaseAdapter<Movie>() {
```

Comment:
I would suggest to remove `BaseAdapter` to something more connected with the Recycler. `Adapter` can has too many meanings in scope of Android framework and Clean Architecture wording as well - it can be done in a separate PR

Reply:
I created issue for this #57